program/rust/src/accounts/price.rs

```diff
@@ -60,6 +62,95 @@ pub struct PriceAccount {
     pub comp_:              [PriceComponent; PC_COMP_SIZE as usize],
 }
 
+/// This is the new version of the PriceAccount.
+/// It will go live once we resize all the price accounts.
+/// It will include more publisher spots and will store cumulative sums to compute TWAPS.
```

Comment:
can you provide more context for this? tell the whole story "we are out of space in price account and in the process of reallocating the accounts...'

Reply:
also like "while both versions of these account structs are available, all operations on price accounts *must* work when passed either version. Operations may check the size of the account to determine which version of the account they have."